### PR TITLE
Initial config for COUNTRY=haiti

### PR DESCRIPTION
**Pull request by wadhwamatic**
### Description

Initial setup for PRISM Haiti demo

## How to test the feature:

- [ ]
- [ ]

## Checklist - did you ...

Test your changes with

- [ ] `REACT_APP_COUNTRY=rbd yarn start`
- [ ] `REACT_APP_COUNTRY=cambodia yarn start`
- [ ] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

File: frontend/src/config/jordan/prism.json [hunk 1]
```diff
@@ -24,6 +24,9 @@
     "admin2_boundaries"
   ],
   "categories": {
+    "drought": {
+      "drought_monitoring": ["cdi"]
```
Comment (ericboucher) [inline frontend/src/config/jordan/prism.json:28]: is this meant to be in this PR?

File: frontend/src/config/haiti/layers.json [hunk 2]
```diff
@@ -0,0 +1,3668 @@
...
+        "value": 12,
+        "label": "> +12C",
+        "color": "#67000d"
+      }
+    ]
+  },
+  "vac_food_insec": {
+    "title": "Food insecurity prevalence (ZimVAC 2020)",
+    "type": "admin_level_data",
+    "path": "data/zimbabwe/zim_vac_food_insec_2020.json",
```
Comment (ericboucher) [inline frontend/src/config/haiti/layers.json:3120]: @wadhwamatic there are a few layer definitions referencing Zimbabwe files in here. It's probably worth removing them for now to clean up. Or would you like to keep them as examples?

**Review by ericboucher — APPROVED** [review]
@wadhwamatic looks good, I fixed an issue with a boundary. I just left a comment regarding references to Zimbabwe layers/files but it's not a blocker. I was able to load all the layers available in the panel.

File: frontend/src/config/haiti/layers.json (hunk 2 above)
Comment (wadhwamatic) [reply frontend/src/config/haiti/layers.json:3120]: was a copy/paste job. I'll remove them

File: frontend/src/config/jordan/prism.json (hunk 1 above)
Comment (wadhwamatic) [reply frontend/src/config/jordan/prism.json:28]: there by mistake, will remove

**Comment by wadhwamatic** [comment]
> @wadhwamatic looks good, I fixed an issue with a boundary. I just left a comment regarding references to Zimbabwe layers/files but it's not a blocker. I was able to load all the layers available in the panel.

what was the boundary issue? and how did you find it / address it?

**Review by ericboucher — APPROVED** [review]

**Pull request merged**